### DISCUSSION TITLE
Serilog.Sinks.Seq bufferBaseFilename file not deleting

### DISCUSSION THREAD

**Discussion by Genex36** [datalust/seq-tickets#1572] [Help] [answered]
I'm implementing SEQ with serilog with the file buffer way.
logs are being sent to my SEQ local instance, but the local file keeps growing.
when does it get deleted? I was imagining that the file would be used as a queue. so append a new one and delete the one above when sent.
I also tried to use bufferSizeLimitBytes to see if it does something but no new files are created and the file just keeps growing.

can anyone help me?
Thanks!

Serilog config
![image](https://user-images.githubusercontent.com/55554440/169010508-9cd46aa3-2a87-4b7c-ab9e-891fe50e048d.png)

**Comment by nblumhardt** [comment, marked as answer]
Hi! The files roll every day, or at 100 MB:

https://github.com/datalust/serilog-sinks-seq/blob/dev/src/Serilog.Sinks.Seq/Sinks/Seq/Durable/DurableSeqSink.cs#L61

The log shipper keeps the two most recent files:

https://github.com/datalust/serilog-sinks-seq/blob/dev/src/Serilog.Sinks.Seq/Sinks/Seq/Durable/HttpLogShipper.cs#L193

Hope this helps,
Nick
